example/App.tsx

```diff
@@ -182,11 +185,55 @@ class RewaredTest implements Test {
   }
 }
 
+class AdConsentTest implements Test {
+  getPath(): string {
+    return 'Consent Form';
+  }
+
+  getTestType(): TestType {
+    return TestType.Interactive;
+  }
+
+  render(onMount: (component: any) => void): React.ReactNode {
+    return (
+      <View style={styles.testSpacing} ref={onMount}>
+        <Button
```

Comment:
A shame about the constraint that it just won't work with the test ad ids (seems like an oversight really, you can't fully exercise something really important!) but I'm okay with a test existing as long as it gets something like 
```suggestion
        <Text>This will not work with the test App ID. For this to work you must configure your real App ID and The UMP has to be configured in the AdMob/Ad Manager backend for the test case to work.</Text>
        <Button
```

Reply:
I guess that'll require the Text import up top as well